static/app/types/profiling.d.ts

```diff
@@ -56,6 +56,7 @@ namespace Profiling {
   type Schema = {
     name: string;
     activeProfileIndex?: number;
+    duration_ns: number;
```

Comment:
Slightly annoyed at the mix of snake case and camel case here.

Reply:
True, cc @phacops here